packages/salesforcedx-sobjects/src/describe/sobjectDescribe.ts

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  CliCommandExecutor,
+  CommandOutput,
+  SfdxCommandBuilder
+} from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+
+export interface SObject {
```

Comment:
How did you arrive at this interface? Is it complete? Is it partially generated?
How would we update this in the future?

Reply:
I used json2ts with the results of the sobjectDescribe call. It's the complete json output, but with result renamed to SObject. We only need the name and Field[] values for the purposes of generation.

Reply:
@DatGuyJonathan - Is json2ts the one that you used too for your debugger? Sorry, I know I asked a similar question earlier but the search feature in the PR in GitHub didn't allow me to find it.